tests/scream_run_integration/test_scream_runtime.py

```diff
@@ -0,0 +1,47 @@
+import yaml
+from scream_run import ScreamConfig
+import os
+import contextlib
+import subprocess
+import pytest
+
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+config_file = os.path.join(dirname, "test.yaml")
```

Comment:
Is this supposed to be using the config file at `/tmp/scream_run_integration/test.yaml` or at `/tmp/test.yaml`? It seems like it should be using the second one since that's the output of `prepare-config`, but I think it's using the first one? Guess I'm having trouble following the intent of this test.

Reply:
Great catch, it should be using the output of `prepare-config` so `/tmp/test.yaml`. But the test still passed in this case because nothing in runtime option got changed during `prepare-config`. `Updating `config_file` path to take in the output from `prepare-config`